specs/core/0_fork-choice.md

```diff
@@ -130,13 +154,43 @@ def get_head(store: Store) -> Hash:
         head = max(children, key=lambda root: (get_latest_attesting_balance(store, root), root))
 ```
 
+#### `should_update_justified_checkpoint`
+
+```python
+def should_update_justified_checkpoint(store: Store, new_justified_checkpoint: Checkpoint) -> bool:
```

Comment:
A reference to the bounce attack, or description of why we have the `best_justified_checkpoint` would be nice for the spec.